src/features/session_replay/aggregate/index.js

```diff
@@ -248,6 +249,7 @@ export class Aggregate extends AggregateBase {
     this.hasSnapshot = false
     this.hasError = false
     this.payloadBytesEstimation = 0
+    if (this.recording) this.timestamp = Date.now()
```

Comment:
This is used to link relative events together. Ie grab the error that occurred during a SR block.  I have an error timestamp, so use that to grab the closest SR chunk with a timestamp before the error timestamp.  Do you think skew would have an impact there since they are all recording on the same system?

Reply:
Yeah, I see no issues as long as it's used to correlate activity all on the same clock.